challenge.md

```diff
@@ -32,19 +32,21 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+
+ This is an absolute path because it is a complete path from start of the filesystem. It also is leading with a /.
+
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+We would use an absolute path becasuse the font is being hosted from a source that is not saved on your computer.
```

Comment:
Hi Matt,

I just pushed up the git push the Navigating the Filesystem from yesterday.
Please let me know if I did not do this correctly again.

Thanks for all your help!

Lauren

On Tue, Sep 29, 2015 at 10:53 AM, Matt Brendzel <notifications@github.com>
wrote:

> In challenge.md
> <https://github.com/ga-wdi-boston/unix-cli-intro-assessment/pull/26#discussion_r40681514>
> :
>
> >   <!-- Answer Ends Here -->
> >
> >  2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
> >
> >   <!-- Answer Starts Here -->
> > -
> > +We would use an absolute path becasuse the font is being hosted from a source that is not saved on your computer.
>
> No problem. Come and find me. Alternatively, if you figure it out
> yourself, feel free to resubmit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/unix-cli-intro-assessment/pull/26/files#r40681514>
> .
>